BridgeAppSDK/DataArchiving/SBAActivityArchive.swift

```diff
@@ -48,15 +60,13 @@ class SBAActivityArchive: SBADataArchive {
                                 archive.insertDataIntoArchive(dataResult, filename: archiveableResult.filename)
                             } else {
                                 let className = NSStringFromClass(archiveableResult.result.classForCoder)
-                                fatalError("Unsupported archiveable result type: \(className)")
+                                assertionFailure("Unsupported archiveable result type: \(className)")
                             }
                         }
                     }
                 }
                 
-                if let error = archive.completeArchive() {
-                    print("Completing the archive of \(activityResult.schemaIdentifier) failed:\n\(error)")
-                } else {
+                if archive.completeArchive() == nil {
```

Comment:
Nit: Could *our* version change the method signature to return a BOOL with success? This is confusing.